configuration/configuration_reference.md

```diff
@@ -519,6 +548,54 @@ lqiao=TfkgShslgJepX80c2JY1trwEskT
 
 [top](#top)
 
+## &lt;authConfigs&gt; {#authConfigs}
+
+Using `<authConfigs>` element GoCD administrators can provide one or more authorization configurations to connect with the authorization server(s).
+
+### Example
+
+```xml
+<authConfigs>
+    <authConfig id="file-auth-config" pluginId="cd.go.authentication.passwordfile">
+    ...
+    </authConfig>
+    <authConfig id="ldap" pluginId="cd.go.authentication.ldap">
+    ...
+    </authConfig>
+</authConfigs>
+```
+
+[top](#top)
+
+## &lt;authConfig&gt; {#authConfig}
+
+`<authConfig>` specifies the [configuration](#property) to be used by the authorization plugin. This will usually allow administrators to configure the connection settings for your authorization plugin, and may include configuration like URLs and credentials, among others.
```

Comment:
> `<authConfig>` specifies the [configuration](#property) to be used by the authorization plugin

#### Change it to

The `<authConfig>` specifies the [configuration](#property) to be used by the authorization plugin...